1_clusters/cloudbuild-destroy.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 3600s # 1-hour
+tags:
+  - modern-cicd-anthos
+  - modern-cicd-clusters
+steps:
+- name: 'gcr.io/${PROJECT_ID}/anthos-platform-installer'
+  id: 'destroy-clusters'
+  entrypoint: 'bash'
+  args:
+  - '-xe'
+  - '-c'
+  - |
+    sed -i "s/YOUR_PROJECT_ID/${PROJECT_ID}/g" terraform.tfvars
+    sed -i "s/YOUR_PROJECT_ID/${PROJECT_ID}/g" backend.tf
+
+    terraform init
+    terraform destroy -auto-approve
+
+- name: 'gcr.io/${PROJECT_ID}/anthos-platform-installer'
+  id: 'remove-attestors-attestations'
+  entrypoint: 'bash'
+  args:
+  - '-xe'
+  - '-c'
+  - |
+    if [ $(gcloud services list --filter="name~binaryauthorization.googleapis.com" --format="value(name)" | wc -w) -gt 0 ]; then
```

Comment:
Minor note: we should move this into a script outside the Cloud Build config so it can easily be embedded/used in separate CI tools.

Alternatively, we should find a way to do this from Terraform itself.

Reply:
Yes, but I wanted to keep the scope of this PR down to moving parts around, not enhancing functionality.  The current destroy uses this script in-line.  We should open a new issue to move this to a script.